app/src/main/java/com/kms/cura/view/PatientProfileViewActivity.java

```diff
@@ -0,0 +1,26 @@
+package com.kms.cura.view;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.Gravity;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.kms.cura.R;
+
+public class PatientProfileViewActivity extends AppCompatActivity {
+    private Toolbar tbPatientProfileView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
The method onCreate is too cumbersome. Should separate it into sub-methods for different purpose ( initControl, addingListeners,...)